charts/airbyte-server/templates/deployment.yaml

```diff
@@ -230,12 +230,15 @@ spec:
   {{ toYaml .Values.extraVolumeMounts | nindent 8 }}
         {{- end }}
       {{- if .Values.extraContainers }}
-      {{ toyaml .Values.extraInitContainers | indent 8 }}
+      {{ toYaml .Values.extraContainers | indent 8 }}
+      {{- end }}
+      {{- if .Values.global.extraContainers }}
+      {{ toYaml .Values.global.extraContainers | indent 8 }}
       {{- end }}
       volumes:
       - name: gcs-log-creds-volume
         secret:
-          secretName: {{ .Release.Name }}-airbyte-gcs-log-creds
+          secretName: {{ ternary (printf "%s-gcs-log-creds" ( .Release.Name )) (.Values.global.credVolumeOverride) (eq .Values.global.deploymentMode "oss") }}
       {{- if .Values.extraVolumes }}
 {{ toYaml .Values.extraVolumes | nindent 6 }}
       {{- end }}
```

Comment:
nit for newlines.  
POSIX sees newline characters as a type of semicolon denoting line ending so it's important for unix tools.  
and it makes `cat` look pretty